src/screens/CurrencySelectModal.js

```diff
@@ -325,10 +325,12 @@ export default function CurrencySelectModal() {
               onChangeText={setSearchQuery}
               ref={searchInputRef}
               searchQuery={searchQuery}
-              {...(ios ? {
-                onFocus: handleFocus,
-                ref: searchInputRef,
```

Comment:
this is carry over from https://github.com/rainbow-me/rainbow/pull/2721 yeah?

Reply:
i think so, lint was failing for me even tho this branch comes from develop 🤷